website/views/shared/profiles/achievements.jade

```diff
@@ -146,8 +146,8 @@ div(ng-if=':: profile.achievements.originalUser')
   
 div(ng-if='profile.achievements.challenges || user._id == profile._id')
   // This is a very strange icon to use. revisit
-  .achievement.achievement-karaoke(ng-show='profile.achievements.challenges')
-  div(ng-class='{muted: !profile.achievements.challenges}')
+  .achievement.achievement-karaoke(ng-show='profile.achievements.challenges.length')
```

Comment:
While we're making changes anyway, can we make this `ng-show` an `ng-if` instead?